src/pkg/scan/job.go

```diff
@@ -146,6 +146,15 @@ func (j *Job) Run(ctx job.Context, params job.Parameters) error {
 	// Get logger
 	myLogger := ctx.GetLogger()
 
+	// shouldStop checks if the job should be stopped
+	shouldStop := func() bool {
+		if cmd, ok := ctx.OPCommand(); ok && cmd == job.StopCommand {
+			return true
```

Comment:
add a log to mention the job is cancelled?

Reply:
> add a log to mention the job is cancelled?

we may not want to overwhelm the core.log with "stop scan job" message in the large scale scenario.